Exec/science/flame_wave/ci-benchmarks/job_info_params.txt

```diff
@@ -0,0 +1,249 @@
+    castro.limit_fluxes_on_small_dens = 0
+    castro.speed_limit = 0
+[*] castro.do_sponge = 1
+    castro.sponge_implicit = 1
+    castro.ext_src_implicit = 0
+    castro.source_term_predictor = 0
+    castro.first_order_hydro = 0
+    castro.xl_ext_bc_type = -1
+    castro.xr_ext_bc_type = -1
+[*] castro.yl_ext_bc_type = 1
+    castro.yr_ext_bc_type = -1
+    castro.zl_ext_bc_type = -1
+    castro.zr_ext_bc_type = -1
+    castro.hse_zero_vels = 0
+[*] castro.hse_interp_temp = 1
+    castro.hse_fixed_temp = -1e+200
+[*] castro.hse_reflect_vels = 1
+[*] castro.fill_ambient_bc = 1
+[*] castro.ambient_fill_dir = 1
+[*] castro.ambient_outflow_vel = 1
+    castro.clamp_ambient_temp = 0
+    castro.ambient_safety_factor = 1.1
+    castro.ambient_density = -1e+200
+    castro.ambient_temp = -1e+200
+    castro.ambient_energy = -1e+200
+    castro.sdc_order = 2
+    castro.sdc_quadrature = 0
+    castro.sdc_extra = 0
+    castro.sdc_solver = 1
+    castro.use_axisymmetric_geom_source = 1
+    castro.add_sdc_react_source_to_advection = 1
+    castro.hydro_memory_footprint_ratio = -1
+    castro.fixed_dt = -1
+    castro.initial_dt = -1
+    castro.dt_cutoff = 1e-12
+    castro.max_dt = 1e+200
+    castro.cfl = 0.8
+[*] castro.init_shrink = 0.1
+    castro.change_max = 1.1
+    castro.check_dt_before_advance = 1
+    castro.check_dt_after_advance = 1
+    castro.plot_per_is_exact = 0
+    castro.small_plot_per_is_exact = 0
+    castro.use_retry = 1
+    castro.retry_subcycle_factor = 0.5
+    castro.retry_small_density_cutoff = -1e+200
+    castro.abundance_failure_tolerance = 0.01
+    castro.abundance_failure_rho_cutoff = -1e+200
+    castro.use_post_step_regrid = 0
+    castro.max_subcycles = 10
+    castro.sdc_iters = 2
+    castro.stopping_criterion_field = 
+    castro.stopping_criterion_value = 1e+200
+    castro.dtnuc_e = 1e+200
+    castro.dtnuc_X = 1e+200
+    castro.dtnuc_X_threshold = 0.001
+[*] castro.do_react = 1
+[*] castro.react_T_min = 6e+07
+    castro.react_T_max = 1e+200
+[*] castro.react_rho_min = 100
+[*] castro.react_rho_max = 5e+07
+    castro.disable_shock_burning = 0
+    castro.T_guess = 1e+08
+    castro.drive_initial_convection = 0
+    castro.drive_initial_convection_tmax = 1e+200
+    castro.drive_initial_convection_reinit_period = 1e+200
+[*] castro.do_grav = 1
+    castro.moving_center = 0
+[*] castro.grav_source_type = 2
+[*] castro.do_rotation = 1
+    castro.bndry_func_thread_safe = 1
+    castro.grown_factor = 1
+    castro.star_at_center = -1
+    castro.do_scf_initial_model = 0
+    castro.scf_maximum_density = -1e+06
+    castro.scf_equatorial_radius = -1e+09
+    castro.scf_polar_radius = -1e+09
+    castro.scf_relax_tol = 0.001
+    castro.scf_max_iterations = 30
+    castro.do_special_tagging = 0
+    castro.max_tagging_radius = 10
+[*] castro.verbose = 1
+    castro.dump_old = 0
+    castro.domain_is_plane_parallel = 0
+    castro.print_update_diagnostics = 0
+[*] castro.sum_interval = 0
```

Comment:
`print_update_diagnostics` defaults to 1 in debug mode (note that it's still marked as modified), and `sum_interval` is set to 1 on the command-line.
```suggestion
[*] castro.print_update_diagnostics = 1
[*] castro.sum_interval = 1
```